### DISCUSSION TITLE
Do we miss a roadmap?

### DISCUSSION THREAD

**Discussion by frederikhors** [orbitinghail/sqlsync#52] [Ideas]
This project is amazing! Really!

But I think we miss a roadmap.

Can we open an issue to pin to the top to collect ideas about the immediate and long future?

Thank you again for your work.

**Comment by carlsverre** [comment]
Hey @frederikhors - do you have any requests to add to the roadmap? Or is this just to have a place for ideas? If the latter, this discussion forum seems like a great place for folks like yourself to introduce new ideas for SQLSync.

**Reply by frederikhors** [reply]
Yeah. I'm talking about something like: "Right now it is not possible to use Svelte, Vue and so on. We'll work on extract a `core` package and on create adapters for them."

Something like that maybe in Readme directly so people can better understand "where we are" and start thinking about adoption.

**Reply by carlsverre** [reply]
Ah, I see what you mean. I misunderstood. Hope this answers your questions! https://github.com/orbitinghail/sqlsync/discussions/53
